opensfm/src/map/python/pybind.cc

```diff
@@ -178,7 +182,8 @@ PYBIND11_MODULE(pymap, m) {
                     &map::Shot::SetShotMeasurements,
                     py::return_value_policy::reference_internal)
       // .def_readwrite("metadata", &map::Shot::shot_measurements_)
-      .def_property("pose", &map::Shot::GetPose, &map::Shot::SetPose)
+      // .def_property("pose", &map::Shot::GetPose, &map::Shot::SetPose)
+      .def_property("pose", (const geometry::Pose& (map::Shot::*) () const)  &map::Shot::GetPose, &map::Shot::SetPose)
```

Comment:
What's the difference ?

Reply:
The difference is that I have a const and non-const getter for the pose and this specifies the const getter